docs/execution/ExecutionTwists.md

```diff
@@ -84,10 +84,11 @@ _If a job fails with a retryable failure (e.g is preempted), it will **not** be
 
 ## Restart
 
-When Cromwell is run in server mode, it is likely that the server will have to be stopped and started again at some point (for example to upgrade Cromwell to a new version).
-In such case, if the `workflow-restart` configuration option is set to `true` (default), Cromwell will restart all jobs that were in progress. Specifically Cromwell will attempt to reconnect to all running jobs of a workflow.
+When Cromwell is run in server mode it is likely that the server will have to be stopped and started again at some point (for example to upgrade Cromwell to a new version).
+In such cases Cromwell will reconnect to all workflows that were in progress and attempt to reconnect to all running jobs of a workflow. Note that because a workflow
+does not "belong" to any one Cromwell instance (it belongs to the cluster), a different instance in a horizontal cluster might reconnect to the workflow instead of the original.
```

Comment:
```suggestion
When Cromwell restarts (for example to upgrade to a new version) it will reconnect to all workflows that were in progress. On the Google and HPC backends only, Cromwell will additionally attempt to reconnect to all running jobs. Note that a workflow
does not "belong" to any one Cromwell instance (it belongs to the cluster), so a different instance in a horizontal cluster might reconnect to the workflow instead of the original.
```